controllers/dcrclient.go

```diff
@@ -492,114 +409,6 @@ func (w *walletSvrManager) connected() ([]*wallettypes.WalletInfoResult, error)
 	return response.walletInfo, response.err
 }
 
-// syncTickets is called when checkForSyncness has returned false and the wallets
-// PoolTickets need to be synced due to manual addtickets, or a status is off.
-// If a ticket is seen to be valid in 1 wallet and invalid in another, we use
-// addticket rpc command to add that ticket to the invalid wallet.
-func (w *walletSvrManager) syncTickets(spuirs []*wallettypes.StakePoolUserInfoResult) error {
```

Comment:
Just to highlight this syncTickets behaviour is removed by this PR. I believe this to be acceptable for two reasons:
- This code was never actually executed because `takeFirstResponse` was always hardcoded true (`w.StakePoolUserInfo(multisig, true)`)
- Sync-ness checks and recovery behaviour exist in another place (see below)
https://github.com/decred/dcrstakepool/blob/32d75352ba2290c33767151205a2de6e26df7b15/controllers/dcrclient.go#L937
https://github.com/decred/dcrstakepool/blob/32d75352ba2290c33767151205a2de6e26df7b15/controllers/dcrclient.go#L1057-L1058